public/editor/stylesheets/editor.less

```diff
@@ -393,7 +381,8 @@ body {
   }
 }
 
-#editor-pane-nav-options-menu:after {
+
+.arrow-tip {
   content: '';
```

Comment:
Let's remove the "content" rule, we only needed that for the ``:after`` pseudo-element